c/termination-memory-linkedlists/cll_search-alloca_true-termination.c

```diff
@@ -40,12 +40,12 @@ void search (node_t* head, int i)
   }
 }
 
-
 int main ()
 {
-  int n = __VERIFIER_nondet_int();
+  int n = abs(__VERIFIER_nondet_int()) + 1;
+  int m = abs(__VERIFIER_nondet_int()) % n;
```

Comment:
What is the intention here? There are multiple overflows here, where I have no idea how termination-proving tools handle them at the moment (even when overflows are ignored): abs(INT_MIN)==INT_MIN, thus defying the purpose of obtaining a positive number (should that have been the idea). abs(MAX_INT)+1 has an overflow.